### PR TITLE
DM-28717: Allow for Stamps formatter to handle bbox

### PR DESCRIPTION
**Pull request by MorganSchmitz**
One of four PRs for [DM-28717](https://jira.lsstcorp.org/browse/DM-28717), the others being afw, obs_base and meas_algorithms.

### CODE REVIEW

File: python/lsst/daf/butler/configs/storageClasses.yaml [hunk 1]
```diff
@@ -184,8 +184,14 @@ storageClasses:
     pytype: lsst.verify.Measurement
   Stamps:
     pytype: lsst.meas.algorithms.Stamps
+    parameters:
+      - bbox
+      - origin
```
Comment (SimonKrughoff) [inline python/lsst/daf/butler/configs/storageClasses.yaml]: The reader seems to only do something with the `bbox` parameter.  What am I missing?

File: python/lsst/daf/butler/configs/storageClasses.yaml [hunk 2]
```diff
@@ -184,8 +184,14 @@ storageClasses:
...
   BrightStarStamps:
     pytype: lsst.meas.algorithms.brightStarStamps.BrightStarStamps
+    parameters:
+      - bbox
+      - origin
```
Comment (SimonKrughoff) [inline python/lsst/daf/butler/configs/storageClasses.yaml]: Same as above

**Review by SimonKrughoff — APPROVED** [review]
Just a clarifying question

File: python/lsst/daf/butler/configs/storageClasses.yaml (hunk 1 above)
Comment (MorganSchmitz) [reply python/lsst/daf/butler/configs/storageClasses.yaml]: Hah! Thanks for asking. I'd just stupidly copied from Image. Removed and tested!

**Pull request merged**